conceptual_model/README.md

```diff
@@ -0,0 +1,32 @@
+Data model
+==========
+
+
+Principles
+----------
+
+One goal is to be able to share data about the identity of
+a transportation user.
+
+For each claim we want to be able to know the data source, and
+if it has been certified.
+
+We defined for this purpose special data types, each with the following 
+attributes :
+
+| Name              | Type       | Description
+| ------------------|------------|---------------------------------
+| value             | any        | The claim value
+| source            | string     | Source of the data
+| certificationDate | datetime   | If the data was certified, and when.
```

Comment:
Can you specify more precisely the field _certificationDate_ ? Is the date when the data is obtained from the source ? Or is a date indicated explicitely by the source ? There is also a _validityDate_ or _expirationDate_

Reply:
Can you please define validityDate and expirationDate ? I think a pull request might be relevant so that you can make a proposal